pkg/utils/statsutils/metrics/metrics.go

```diff
@@ -13,11 +15,16 @@ import (
 	"github.com/solo-io/solo-kit/pkg/api/v1/resources"
 	"github.com/solo-io/solo-kit/pkg/api/v1/resources/core"
 	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
 	"go.opencensus.io/tag"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/util/jsonpath"
 )
 
+const (
+	ClearStatusMetricsEnvVar = "GLOO_CLEAR_STATUS_METRICS"
```

Comment:
Do we want this to be prefixed with GLOO given upcoming naming changes or do we think thats in fact a boon?

Im currently leaning towards this being the right naming but figure i might as well make the though persisted so others can chime in